src/components/CvViewer/CvViewer.types.ts

```diff
@@ -8,38 +8,46 @@ export enum Template {
   ModernColumnBlack = 'modern column black'
 }
 
+export type PersonalInfoType = {
+  jobTitle?: string;
+  fullName?: string;
+  email?: string;
+  address?: string;
+  avatar?: string;
+  isShowAvatar?: boolean;
+  description?: string;
+};
+
+export type WorkExperienceType = {
+  id: string;
+  companyName?: string;
+  position?: string;
+  dateTime?: string[]; // american date format
+  startDate?: string; // american date format
+  endDate?: string; // american date format
+  isCurrentWork?: boolean;
+  description?: string;
+  remark?: string;
+};
+
+export type EducationType = {
+  id: string;
+  universityName?: string;
+  speciality?: string;
+  dateTime?: string[]; // american date format
+  startDate?: string; // american date format
+  endDate?: string; // american date format
+  isCurrentEducation?: boolean;
+  description?: string;
+  remark?: string;
```

Comment:
Here's the same, remark => description, or maybe just get rid of remark, because I see that we already have description here

Reply:
Add to description some AI prefix/suffix

Reply:
please see the comments above